api/controllers/v0.3/UsernameValidationAPI.js

```diff
@@ -1,19 +1,18 @@
 /* eslint-disable */
-import Card from "../../models/v0.3/modelCard";
 import IlsHelper from "./ILSHelper";
 
 /**
  * A class that validates usernames against the ILS.
  */
 const UsernameValidationAPI = () => {
   // class IntegrationError < StandardError; end
-  const USERNAME_PATTERN = /\A[a-zA-Z0-9]{5,25}\z/;
+  const USERNAME_PATTERN = /^[a-zA-Z0-9]{5,25}$/;
   const AVAILABLE_USERNAME_TYPE = "available-username";
   const UNAVAILABLE_USERNAME_TYPE = "unavailable-username";
   const INVALID_USERNAME_TYPE = "invalid-username";
 
-  const STANDARD_CARD_TYPE = Card.STANDARD_CARD_TYPE;
-  const TEMPORARY_CARD_TYPE = Card.TEMPORARY_CARD_TYPE;
+  const STANDARD_CARD_TYPE = "standard";
```

Comment:
Why the shift away from the `Card` constants?

Reply:
Didn't want the cyclical dependencies. The Card model imports this class.